users/signals.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from users.models import Profile
+
+
+@receiver(post_save, sender=User)
+def create_profile(sender, instance, created, **kwargs):
+    if created:
+        Profile.objects.create(user=instance)
```

Comment:
Might as well integrate this directly into the view logic, instead of using signals?

Reply:
For me it's more clear to do something like this. In this file you can see and keep track all signals handler (for example if we have more models which are associated with User models), instead of looking in different function implementation in views.py file.

Reply:
All right.